src/VisualStudio/IntegrationTests/AbstractInteractiveWindowTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.VisualStudio.Test.Utilities;
+using Roslyn.VisualStudio.Test.Utilities.OutOfProcess;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests
+{
+    public abstract class AbstractInteractiveWindowTest : AbstractIntegrationTest
+    {
+        private const string Edit_SelectionCancelCommand = "Edit.SelectionCancel";
+
+        private static readonly char[] LineSeparators = { '\r', '\n' };
+
+        protected readonly CSharpInteractiveWindow_OutOfProc _interactiveWindow;
```

Comment:
>_interactiveWindow [](start = 61, length = 18)

Protected vnarciables should be pascal cased.  

Going to stop adding this comment.  Please just assume it for every field.